.gitignore

```diff
@@ -1,11 +1,32 @@
+/ENIGMA/
+/emake
+/test-runner
+/LinuxExtra/
+/MacOS/build/Release/EnigmaXcode.app/Contents/Info.plist
+/MacOS/build/Release/EnigmaXcode.app/Contents/Resources/*
+
+/ENIGMAsystem/SHELL/API_Switchboard.h
+/ENIGMAsystem/SHELL/Preprocessor_Environment_Editable/
+/ENIGMAsystem/SHELL/ENIGMAengine
+/ENIGMAsystem/SHELL/.objs
+/ENIGMAsystem/additional/
+/ENIGMAsystem/Additional/
+
+.deps
+.eobjs/
+/plugins/
+/CompilerSource/bin/
+/CompilerSource/JDI/dox/
+/CompilerSource/JDI/test/defines_linux.txt
+/CompilerSource/JDI/test/defines_mingw.txt
+/CompilerSource/stupidity-buffer/obj
+
+.DS_Store
+
+ValgrindOut.xml
 hs_err_pid*.log
-searchdirs.txt
-redirfile.txt
-definitions.h
```

Comment:
lgm still writes definitions.h to enigma's root. @RobertBColton should prolly fix it to write to enigmas workdir

Reply:
Resolved.